src/commands/CmdInfo.cpp

```diff
@@ -374,7 +374,7 @@ int CmdInfo::execute(std::string& output) {
     // the context.columns map.
     for (auto& att : all) {
       if (att.substr(0, 11) != "annotation_" && att.substr(0, 5) != "tags_" &&
-          att.substr(0, 4) != "dep_" &&
+          att.substr(0, 4) != "dep_" && att.substr(0, 4) != "tag_" &&
```

Comment:
At one point we accidentally used `tags_` instead of `tag_`, and I think fixing that bug was what introduced #3607. please remove the `"tags_"` check here.

Reply:
Got it. Updated the PR.